internal/pkg/template/templates/environment/partials/cdn-resources.yml

```
@@ -44,9 +44,23 @@ UniqueAliasesAction:
   Properties:
     ServiceToken: !GetAtt UniqueJSONValuesFunction.Arn
     Aliases: !Ref Aliases
+    {{- if .CDNConfig.Static}}
+    AdditionalAlias: {{.CDNConfig.Static.Alias}}
+    {{- end}}
     FilterFor: !Ref ALBWorkloads
 {{- end}}{{/* endif or .CDNConfig.ImportedCertificate .DelegateDNS */}}
 
+{{- if .CDNConfig.Static}}
+CloudFrontOriginAccessControl:
+  Type: AWS::CloudFront::OriginAccessControl
+  Properties: 
+    OriginAccessControlConfig: 
+        Description: An optional description for the origin access control
```

(I know this was unintentional, just throwing out an idea)
```suggestion
        Description: Access control for static s3 origin```